index.html

```diff
@@ -175,6 +175,178 @@ <h5 class="сustomer-кeviews">Отзывы</h5>
+                        <p class="payment-method">Способ оплаты</p>
+                    </div>
+                    <div class="make-order-location-box">
+                        <p class="city-set">Укажите Город</p>
+                        <select class="city-select" name="">
+                        <option value="1">Мозырь</option>
+                        <option value="2">Брест</option>
+                        <option value="3">Минск</option>
+                        <option value="4">Гомель</option>
+                        <option value="5">Витебск</option>
+                        <option value="6">Могилев</option>
+                     </select>
+                        <input class="city-add" type="text" placeholder="Добавить город">
+                    </div>
+                </div>
+                <div class="payment-method-box">
+                    <div class="payment-method-item">
+                        <label class="payment-">
+                        <input type="radio" class="input-radio-payment" name="radio-bank-transfer">
+                        <span class="choice-of-payment-method"></span>
+                         Наличными курьеру
+                    </label>
+                    </div>
+                    <div class="payment-method-item">
+                        <label class="label-box">
+                        <input type="radio" class="input-radio-payment" name="radio-bank-transfer">
+                        <span class="choice-of-payment-method" ></span>
+                        Банковский перевод
+                    </label>
+                    </div>
+                    <div class="payment-method-item">
+                        <label class="label-box">
+                        <input type="radio" class="input-radio-payment" name="radio-bank-transfer">
+                        <span class="choice-of-payment-method"></span>
+                        Банковский перевод
+                    </label>
+                    </div>
+                </div>
+                <h1 class="additional-services-text">Дополнительные услуги</h1>
+                <div class="additional-services-wrapper">
+                    <div class="additional-services-item">
+                        <input class="additional-services-checkbox" type="checkbox" id="cheackbox_call">
+                        <label class="additional-services-label" for="cheackbox_call">Вызов замерщика</label>
+                    </div>
+                    <div class="additional-services-item">
+                        <input class="additional-services-checkbox" type="checkbox" id="cheackbox_packaging">
+                        <label class="additional-services-label" for="cheackbox_packaging">Упаковка для хрупких предметов</label>
+                    </div>
+                    <div class="additional-services-item">
+                        <input class="additional-services-checkbox" type="checkbox" id="cheackbox_crane">
```

Comment:
cheackbox
or
checkbox
Maybe you need install ESLint, which will checked you gramma. Read about it.